### ISSUE TITLE
steinbock classify ilastik prepare not working on Ubuntu

### ISSUE THREAD

**Issue by simeonmayala** [BodenmillerGroup/steinbock#79]
I get the following error message when I try to run the `steinbock classify ilastik prepare` command on Ubuntu:

```
root@simeon-Latitude-7490:~# steinbock classify ilastik prepare --cropsize 50 --seed 123
/app/entrypoint.sh: line 3:    13 Killed                  python -m steinbock "$@"
```

**Comment by jwindhager** [comment]
Hi @simeonmayala 

thank you for filing this issue and using steinbock.

This error most likely indicates insufficient system resources, memory (RAM) in particular. How big is the image? How much memory is your system equipped with?

**Comment by jobha** [comment]
We have the same problem on several different computers using Linux and WSL. The file we're trying to process is .95 GB. The process uses 100% of memory (16 GB) before it shuts down with this error.

**Comment by jwindhager** [comment]
Uh-oh, that doesn't sound right, thanks so much for reporting @jobha! Would it be possible to [share](mailto:jonas.windhager@uzh.ch) the file with me so that I can investigate? I'd really appreciate it, since it's a bit hard to reproduce such issues without having access to the data.

**Comment by jwindhager** [comment]
Just tagging @malmstroem here as well

**Comment by jobha** [comment]
Sure, I'll send you the file  @jwindhager

**Comment by jwindhager** [comment]
Just to keep you posted: I have identified the issue and am working to resolve it.

Note to myself: the problem is caused by upsizing the image using [skimage.transform.resize](https://scikit-image.org/docs/dev/api/skimage.transform.html#skimage.transform.resize)
![image](https://user-images.githubusercontent.com/7105759/146750862-5c14093d-e3ec-4182-b413-9cdf25c99c7d.png)

**Comment by jwindhager** [comment]
This is fixed with v0.10.3
